Knoema.Client/Data/TimeSeriesRawData.cs

```diff
@@ -19,10 +19,12 @@ internal void OnDeserialized(StreamingContext context)
 		{
 			Dimensions = _dimensions.Select(pair =>
 			{
+				if (pair.Value.Type != JTokenType.Object || pair.Value["key"] == null || pair.Value["name"] == null)
+					return null;
 				var value = pair.Value.ToObject<DimensionItem>();
```

Comment:
pair.Value as separate variable. Also renaming value variable to item or dimensionItem makes sense, don't you think?

Reply:
done